src/kava-model.js

```diff
@@ -33,6 +33,7 @@ class KavaModel {
   flavorParamsId: number = NaN;
   networkConnectionType: string;
   playerJSLoadTime: ?number = null;
+  userId: string;
```

Comment:
add a getter

Reply:
this is not necessary, it is handled by the delegate so no need to save it here

Reply:
why to save is getters?